### ISSUE TITLE
SRW - exported .py file has incorrect set of options in run_all_reports()

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#191]
The exported `.py` file does not have the same options in the `run_all_reports()` function as in [sirepo/pkcli/srw.py](https://github.com/radiasoft/sirepo/blob/master/sirepo/pkcli/srw.py#L95). Need to fix it.

**Comment by mrakitin** [comment]
An exported example added. It fails to run:
```python
/home/vagrant/.pyenv/shims/python /home/vagrant/src/radiasoft/sirepo/tests/importer_data/exported_undulator_radiation.py
Single-electron SR calculatiton ... completed (lasted 3.52131 s)
Extracting intensity and saving it to a file ... completed (lasted 0.007159 s)
Single-electron SR calculatiton ... completed (lasted 1.317247 s)
Extracting intensity and saving it to a file ... completed (lasted 0.012282 s)
Propagation ... Traceback (most recent call last):
  File "/home/vagrant/src/radiasoft/sirepo/tests/importer_data/exported_undulator_radiation.py", line 319, in <module>
    run_all_reports()
  File "/home/vagrant/src/radiasoft/sirepo/tests/importer_data/exported_undulator_radiation.py", line 316, in run_all_reports
    srwl_bl.SRWLBeamline(_name=v.name, _mag_approx=mag).calc_all(v, op)
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/srwl_bl.py", line 1047, in calc_all
    _fname = os.path.join(_v.fdir, _v.ws_fni) if(len(_v.ws_fni) > 0) else '')
  File "/home/vagrant/.pyenv/versions/2.7.10/lib/python2.7/site-packages/srwl_bl.py", line 713, in calc_wfr_prop
    srwl.PropagElecField(_wfr, self.optics)
RuntimeError: No objects were submitted for parsing

Process finished with exit code 1
```

**Comment by mrakitin** [comment]
If `` is set to `False`, it runs successfully:
```python
/home/vagrant/.pyenv/shims/python /home/vagrant/src/radiasoft/sirepo/tests/importer_data/exported_undulator_radiation.py
Single-electron SR calculatiton ... completed (lasted 3.590955 s)
Extracting intensity and saving it to a file ... completed (lasted 0.007103 s)
Single-electron SR calculatiton ... completed (lasted 1.322022 s)
Extracting intensity and saving it to a file ... completed (lasted 0.01115 s)

Process finished with exit code 0
```
See 1b9aae3.

**Comment by mrakitin** [comment]
Will fix it along with updates for #85.

**Comment by mrakitin** [comment]
Fixed in 56fb6bf.
